frontend/packages/console-shared/src/components/dashboard/utilization-card/TopConsumerPopover.tsx

```diff
@@ -46,12 +47,14 @@ const ConsumerPopover: React.FC<ConsumerPopoverProps> = React.memo(
 
 export default ConsumerPopover;
 
-const getResourceToWatch = (model: K8sKind, namespace: string) => ({
-  isList: true,
-  kind: model.crd ? referenceForModel(model) : model.kind,
-  namespace,
-  prop: 'k8sResources',
-});
+const getResourceToWatch = (model: K8sKind, namespace: string) => [
```

Comment:
Why change the resulting object to a single-element array? (The name of the function is now rather confusing.)

You can still use it like

```diff
-getResourceToWatch(model, namespace)
+[getResourceToWatch(model, namespace)]
```

which is more flexible, IMO. Same question for the similar code below :smiley:

Reply:
right, this is not needed. I've removed the change and using `[getResourceToWatch(model, namespace)]`